postr/schedule/task_processor.py

```diff
@@ -16,7 +16,7 @@
     'discord': discord_api.discord_client,
     'reddit': Reddit(),
     'twitter': Twitter(),
-    'facebook': FacebookChatApi(),
+    # 'facebook': FacebookChatApi(),
```

Comment:
facebook fails with a 'NameError: name 'open' is not defined' message on startup, not sure why, so it's disabled